client/lib/products-values/translations.js

```diff
@@ -133,13 +148,15 @@ export const getJetpackProductsCallToAction = () => {
 		{
 			v1: translate( 'Get Jetpack Search' ),
 			i5: translate( 'Get Site Search' ),
+			spp: translate( 'Get Site Search' ),
 		}[ currentCROvariant ] || translate( 'Get Search' );
 	const scan =
 		currentCROvariant === 'v1' ? translate( 'Get Jetpack Scan' ) : translate( 'Get Scan' );
 	const antiSpam =
-		currentCROvariant === 'v1'
-			? translate( 'Get Jetpack Anti-spam' )
-			: translate( 'Get Anti-spam' );
+		{
+			v1: translate( 'Get Jetpack Anti-spam' ),
+			spp: translate( 'Get Anti-Spam' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.